### ISSUE TITLE
MIGX 2.11.0-pl Please remove new inline styling 

### ISSUE THREAD

**Issue by itsmerobliberty** [Bruno17/MIGX#293]
In the latest MIGX release there is an additional `width: calc(100% - 5px); padding: 0 !important;` on the `x-form-item x-tab-item modx-tv` items within the modal edit window. This inline styling, specifically the padding because of the `!important`, interferes with some custom theme styles.

Would you mind removing the new styling, or at least the `!important` tag so we can apply custom theme styling again?

Thank you.

**Comment by Bruno17** [comment]
seems, this was added with this commit, to reduce to much padding
https://github.com/Bruno17/MIGX/commit/a80a85838780c825e17b15ef21d04123c75966e5

will have a look into it

**Comment by itsmerobliberty** [comment]
Yes line 87 of `core/components/migx/templates/mgr/fields.tpl`. It used to just add a `padding:10px 0 0;` before the last update.

Thanks for looking into it.

**Comment by Bruno17** [comment]
sorry, I was trying to remove the !important, but it doesn't work without and I can't spend much time with this styling-stuff.

**Comment by itsmerobliberty** [comment]
Interesting, in the short term I've removed it on my local installs without any issues. I'll look deeper into your "styling-stuff" and post back if I find a more elegant solution.

**Comment by Bruno17** [comment]
when I remove the !important, the fields get longer, than the panel.

**Comment by c0mat0se** [comment]
I agree that the !important on `x-form-item x-tab-item modx-tv` makes it very difficult for us to implement our own styles. I solved this by removing the padding with the important, and adding the following properties:

`overflow:hidden;
box-sizing: border-box;
width:100%;`

I would prefer to have the important removed in an update, so that I do not have to do it manually every-time I update MIGX.
